docs/tasks/configure-pod-container/configure-pod-initialization.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Configuring Pod Initialization
+---
+
+{% capture overview %}
+This page shows how to use an init Container to initialize a Pod before the
+application Container runs.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Creating a Pod that has an init Container
+
+In this exercise you create a Pod that has one application Container and one
+init Container. The init Container runs to completion before the application
+container starts.
+
+Here is the configuration file for the Pod:
```

Comment:
We should probably a doc about what happens when features move out of alpha.  We should also talk about how alpha features aren't guaranteed to not-break between releases.